core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala

```diff
@@ -361,12 +362,10 @@ case class SnipDirective(page: Page, variables: Map[String, String])
       val (text, snippetLang) = Snippet(file, labels)
       val lang = Option(node.attributes.value("type")).getOrElse(snippetLang)
       val group = Option(node.attributes.value("group")).getOrElse("")
-      new VerbatimGroupNode(text, lang, group, node.attributes.classes).accept(visitor)
-      if (variables.contains(GitHubResolver.baseUrl) &&
-        variables.getOrElse(SnipDirective.showGithubLinks, "false") == "true") {
-        val p = resolvePath(page, Path.toUnixStyleRootPath(file.getAbsolutePath), labels.headOption).base.normalize.toString
-        new ClassyLinkNode(p, "snippet-full-source github", new TextNode("Full source at GitHub")).accept(visitor)
```

Comment:
Shouldn't the `snippet-full-source` class be removed again with this PR?

Reply:
I did not find a mention of `snippet-full-source` anywhere else in Paradox. So it seems it was not even used for styling or anything else.

Reply:
I added it so that I could style the element in the paradox material theme. :+1: for adding something equivalent in the new version.

Reply:
Currently the new links have `class="icon go-to-source"`. Is that OK?